bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1219,878 +1207,382 @@ lnd/clm2/surfdata_map/surfdata_ne120np4_78pfts_CMIP6_simyr1850_c170824.nc</fsurd
->lnd/clm2/mappingdata/maps/0.47x0.63/map_3x3min_MODIS-wCsp_to_0.47x0.63_nomask_aave_da_c170914.nc</map>
-<map frm_hgrid="3x3min"    frm_lmask="USGS"  to_hgrid="0.47x0.63"   to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.47x0.63/map_3x3min_USGS_to_0.47x0.63_nomask_aave_da_c170914.nc</map>
-<map frm_hgrid="5x5min"    frm_lmask="IGBP-GSDP"  to_hgrid="0.47x0.63"   to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.47x0.63/map_5x5min_IGBP-GSDP_to_0.47x0.63_nomask_aave_da_c170914.nc</map>
-<map frm_hgrid="5x5min"    frm_lmask="ISRIC-WISE"  to_hgrid="0.47x0.63"   to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.47x0.63/map_5x5min_ISRIC-WISE_to_0.47x0.63_nomask_aave_da_c170914.nc</map>
+<map frm_hgrid="3x3min"    frm_lmask="nomask"  to_hgrid="0.47x0.63"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/0.47x0.63/map_3x3min_nomask_to_0.47x0.63_nomask_aave_da_c191016.nc</map>
 <map frm_hgrid="5x5min"    frm_lmask="nomask"  to_hgrid="0.47x0.63"   to_lmask="nomask" 
 >lnd/clm2/mappingdata/maps/0.47x0.63/map_5x5min_nomask_to_0.47x0.63_nomask_aave_da_c170914.nc</map>
-<map frm_hgrid="5x5min"    frm_lmask="ORNL-Soil"  to_hgrid="0.47x0.63"   to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.47x0.63/map_5x5min_ORNL-Soil_to_0.47x0.63_nomask_aave_da_c170914.nc</map>
+<map frm_hgrid="0.9x1.25"    frm_lmask="nomask"  to_hgrid="0.47x0.63"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/0.47x0.63/map_0.9x1.25_nomask_to_0.47x0.63_nomask_aave_da_c191016.nc</map>
 
 <!-- mapping files for 0.47x0.63 END --> 
 
 
-<map frm_hgrid="0.5x0.5"  frm_lmask="MODIS"  to_hgrid="0.9x1.25" to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.9x1.25/map_0.5x0.5_landuse_to_0.9x1.25_aave_da_110307.nc</map>
-<map frm_hgrid="0.25x0.25"  frm_lmask="MODIS"  to_hgrid="0.9x1.25"   to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.9x1.25/map_0.25x0.25_MODIS_to_0.9x1.25_nomask_aave_da_c170321.nc</map>
-<map frm_hgrid="0.5x0.5"  frm_lmask="AVHRR"  to_hgrid="0.9x1.25" to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.9x1.25/map_0.5x0.5_lanwat_to_0.9x1.25_aave_da_110307.nc</map>
+<map frm_hgrid="0.5x0.5"  frm_lmask="nomask"  to_hgrid="0.9x1.25" to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/0.9x1.25/map_0.5x0.5_nomask_to_0.9x1.25_aave_da_c121019.nc</map>
+<map frm_hgrid="0.25x0.25"  frm_lmask="nomask"  to_hgrid="0.9x1.25"   to_lmask="nomask" 
+>lnd/clm2/mappingdata/maps/0.9x1.25/map_0.25x0.25_nomask_to_0.9x1.25_nomask_aave_da_c191016.nc</map>
 <map frm_hgrid="10x10min" frm_lmask="nomask" to_hgrid="0.9x1.25" to_lmask="nomask"
 >lnd/clm2/mappingdata/maps/0.9x1.25/map_10minx10min_topo_to_0.9x1.25_aave_da_110630.nc</map>
-<map frm_hgrid="5x5min"   frm_lmask="IGBP-GSDP" to_hgrid="0.9x1.25" to_lmask="nomask"
->lnd/clm2/mappingdata/maps/0.9x1.25/map_5minx5min_soitex_to_0.9x1.25_aave_da_110722.nc</map>
 <map frm_hgrid="5x5min"   frm_lmask="nomask" to_hgrid="0.9x1.25" to_lmask="nomask"
 >lnd/clm2/mappingdata/maps/0.9x1.25/map_5minx5min_irrig_to_0.9x1.25_aave_da_110529.nc</map>
```

Comment:
`map_5minx5min_irrig_to_0.9x1.25_aave_da_110529.nc` which I actually replaced initially with `map_5x5min_nomask_to_0.9x1.25_nomask_aave_da_c191016.nc`  
needs updating because used as a generic nomask file it resulted in differences greater than 1e-13.

In particular my surfdata_0.9x1.25_nomask_78pfts_CMIP6_simyr2000_c191107.namelist points to the following spin-off files:

`map_forganic     = '/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/maps/0.9x1.25/map_5x5min_nomask_fromISRIC_to_0.9x1.25_nomask_aave_da_c191106.nc'`

` map_fsoitex      = '/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/maps/0.9x1.25/map_5x5min_nomask_fromIGBP_to_0.9x1.25_nomask_aave_da_c191107.nc'`

` map_fsoildepth   = '/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/maps/0.9x1.25/map_5x5min_nomask_fromORNL_to_0.9x1.25_nomask_aave_da_c191106.nc'`